engine/schema/src/main/java/org/apache/cloudstack/storage/datastore/db/PrimaryDataStoreDaoImpl.java

```diff
@@ -443,9 +451,14 @@ public List<StoragePoolVO> findDisabledPoolsByScope(long dcId, Long podId, Long
 
     @Override
     public List<StoragePoolVO> findLocalStoragePoolsByTags(long dcId, long podId, Long clusterId, String[] tags, boolean validateTagRule) {
+        return findLocalStoragePoolsByTags(dcId, podId, clusterId, tags, validateTagRule, null);
+    }
+
+    @Override
+    public List<StoragePoolVO> findLocalStoragePoolsByTags(long dcId, long podId, Long clusterId, String[] tags, boolean validateTagRule, String keyword) {
         List<StoragePoolVO> storagePools = null;
         if (tags == null || tags.length == 0) {
```

Comment:
@GaOrtiga will you apply this sugestion (ready-for-merge otherwise)

Reply:
@GutoVeronezi @DaanHoogland

Since no changes have been made to this line, I believe the proposed suggestion would not align with the scope of this PR.
 It would be more appropriate to address this in a separate PR focused on refactoring.